framework/widgets/ActiveField.php

```diff
@@ -315,10 +317,17 @@ public function error($options = [])
      *
      * @return $this the field object itself
      */
-    public function hint($content, $options = [])
+    public function hint($content = null, $options = [])
```

Comment:
@rob006 
As I said, for consistency with `public function label($label = null, $options = [])`. IMO you either do something properly or don't do it at all. Besides, affected code upgrade for this change is super easy.